two_factor/apps.py

```diff
@@ -10,3 +13,21 @@ def ready(self):
         if getattr(settings, 'TWO_FACTOR_PATCH_ADMIN', True):
             from .admin import patch_admin
             patch_admin()
+
+
+@receiver(setting_changed, dispatch_uid="two_factor.handle_setting_changed")
+def handle_setting_changed(sender, setting: str, value, **kwargs):
+    from .admin import patch_admin, unpatch_admin, __default_admin_site__
```

Comment:
why did you put the import here?

Reply:
Circular dependencies, `apps.py` is imported before everything else, and if that pulls in `admin.py` before django is ready, then everything collapses.

Reply:
ok :)